org.x.adriconf.json

```diff
@@ -0,0 +1,171 @@
+{
+   "app-id": "org.x.adriconf",
```

Comment:
See https://github.com/flathub/flathub/pull/605#issuecomment-419066807 is this app-id correct?